src/main.c

```diff
@@ -58,17 +59,15 @@ txn_approve()
   PRINTF("Signing message: %.*h\n", msg_len, msgpack_buf);
 
   cx_ecfp_private_key_t privateKey;
-  algorand_private_key(&privateKey);
+  algorand_key_derive(0, &privateKey);
```

Comment:
Have you tried to use this with U2F on the Nano S on Windows? When we tried deriving the key here, the extra latency of the key derivation broke https://wallet.myalgo.com/ in this use case.

Reply:
add `io_seproxyhal_io_heartbeat` in algorand key derive process as a "workaround" (i.e. to let the sdk u2f stack answer to keep alive requests).

Reply:
Done.